README.md

```diff
@@ -181,26 +187,39 @@ Follow these steps to update the Airflow docker step. Reasons for doing this inc
 - Modifying the `Dockerfile` to upgrade the Airflow version
 - Modifying the haproxy configuration
 
-#### Update Process
+### Update Process
+
+#### Testing a new Airflow version
 
 - Read the "Significant Changes" sections of the Airflow release notes between the versions in question: https://github.com/apache/airflow/releases/
   - Apache Airflow is a very active project, and these release notes are pretty dense. Keeping a regular update cadence will be helpful to keep up the task of updating airflow from becoming an "information overload" job.
-- Snap a backup of the Airflow postgreSQL database
+- Create a local branch with the [Dockerfile](./Dockerfile) modified to the version you intend to test
+- In the [docker-compose.yaml](./docker-compose.yaml), replace `image: atddocker/atd-airflow:production` with `build: .`
+- Build the Docker images locally:
+```shell
+$ docker compose build
```

Comment:
I think it's worth mentioning here the flag `--no-cache` to give people a route to build things anew, which is important for images that include `apt` commands in their Dockerfile.

Reply:
cool, thanks! i just added this.